src/root.jsx

```diff
@@ -252,21 +260,30 @@ class Root extends React.Component {
         const speakerObj = {
             1: record.get('speaker1'),
             2: record.get('speaker2'),
+            3: record.get('speaker3'),
         };
         const speechTitleObj = {
             1: record.get('speechTitle1'),
             2: record.get('speechTitle2'),
+            3: record.get('speechTitle3'),
         };
         const evaluator1 = record.get('evaluator1');
         const evaluator2 = record.get('evaluator2');
         const grammarian = record.get('grammarian');
         const timer = record.get('timer');
         const ahCounter = record.get('ahCounter');
 
+        const speakerTitles = {
```

Comment:
`speakerTitles` is effectively an object constant, so why not just move it to the same outermost scope as other constants (e.g. `SPREADSHEET_ID`), use same uppercase naming scheme, and refer to it directly within `getSpeakerSlotRow`? Right now `speakerTitles` is re-created on each render and passed into `getSpeakerSlotRow`.

Reply:
@borncrusader moving out the const object is a good idea.